scripts/install.sh

```diff
@@ -12,6 +12,8 @@ HAS_CURL="$(type "curl" &> /dev/null && echo true || echo false)"
 HAS_WGET="$(type "wget" &> /dev/null && echo true || echo false)"
 HAS_SYSTEMD="$(type "systemctl" &> /dev/null && echo true || echo false)"
 
+HAS_STRESS_NG="$(type "stress-ng" &> /dev/null && echo true || echo false)"
```

Comment:
Doesn't this check makes stress-ng a necessity for m-agent installation even if the user doesn't want to run any stress-chaos based experiment?

Reply:
That makes sense, I have removed the check from the installation script now. We will be checking for stress-ng as usual in the steady-state only.